Makefile

```diff
@@ -24,11 +24,17 @@ build: fmt vet
 
 .PHONY: build-container
 build-container: fmt vet test
-	podman build -t ${IMG} .
+	podman manifest rm ${IMG} || true && \
+	podman build --build-arg TARGET_ARCH=amd64 --manifest=${IMG} -t ${IMG_REPOSITORY}:amd64-${IMG_TAG} . && \
```

Comment:
Do we really need the additional `$arch-${IMG_TAG}` tags? IMO we can omit them. The manifest keeps track of all images.

Reply:
You are correct that the manifest keeps track of them. However, when we upload to a container registry, these images can appear without any tagging. This could prevent accidental deletion. I can make changes if this is a concern for you.

Reply:
Sounds like a bug in those registries to me if they don't make use of manifests? I'd remove the additional tags.

Reply:
Done.